### PR TITLE
re-arrange program tables to vertical tow-page layout

### PR DESCRIPTION
**Pull request by tyrasd**
Changed program tables to use two-page spanning vertical layout, to allow full titles of talks to fit.

Here's a before/after of the citical day (Sunday, 5 parallel tracks):

 → 

 → 

 → 

[full table rendering](https://github.com/osmfoundation/sotm2019-booklet/files/3493352/master.pdf)

This PR also includes a few additional changes:
* changed fonts to barlow/d-din as on our logo and website
* more prominent dashed lines around talk slots
* moved photo to saturday afternoon break
* slight rewording of breaks / events
* fixed a few typos (speaker name)

### CODE REVIEW

**Comment by tyrasd** [comment]
//cc @rabidllama @Nakaner @ChristineKarch @yair-grinb

**Comment by MarcoMinghini** [comment]
Thanks! This is definitely a very useful proposal that overcomes the problem of changing (shortening) title talks.

File: abstract-commands.tex [hunk 1]
```diff
@@ -230,7 +230,8 @@
 % define colours
 \definecolor{GHS}{cmyk}{0.16 0 .05 0}
 \definecolor{HSO}{cmyk}{0.02 0 0.12 0.11}
-\definecolor{academic}{cmyk}{0 0.02 0.23 0}
+%\definecolor{academic}{cmyk}{0 0.02 0.23 0}
+\definecolor{academic}{cmyk}{0.35 0 0.33 0.16}
```
Comment (Nakaner) [inline abstract-commands.tex:234]: The academic colour was intentionally defined as yellow. However, I accidentially used the normal HSW colour for most of the academic track. I wanted to allow the readers to distinguish the academic track from the other tracks.

File: master.tex [hunk 2]
```diff
@@ -1,19 +1,20 @@
-\documentclass[
-version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany]{scrbook}
+\documentclass[version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany,twoside]{scrbook}
+%\documentclass[twoside]{article}
```
Comment (Nakaner) [inline master.tex:2]: Why did you add this comment?

File: master.tex [hunk 3]
```diff
@@ -1,19 +1,20 @@
-\documentclass[
-version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany]{scrbook}
+\documentclass[version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany,twoside]{scrbook}
```
Comment (Nakaner) [inline master.tex:1]: I though `twoside` isn't necessary because I define the margins manually (I calculated them manually back in 2014), is it?

File: master.tex [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
+%\documentclass[twoside]{article}
 %\pdfminorversion=4
 \usepackage{polyglossia}
 \setdefaultlanguage{english}
 
-\setmainfont{Source Sans Pro}
-\setsansfont{Source Sans Pro}
+\setmainfont{Barlow}
+\setsansfont{Barlow}
+\newfontfamily\numeral{d-din}
```
Comment (Nakaner) [inline master.tex:9]: No changes to the font please. Barlow is a too high font. I choose Source Sans Pro intentionally last year.

DIN font is no font for typesetting texts. It's a font for signs but we aren't producing destination signs for roads here.

File: master.tex [hunk 5]
```diff
@@ -57,7 +58,8 @@
 \usepackage{multirow}
 %\usepackage{booktabs}
 
-\usepackage{array}
+%\usepackage{array}
+\usepackage{array,longtable,color}
```
Comment (Nakaner) [inline master.tex:62]: Why do you add the longtable package? Do we have any tables spreading beyond a page break?

File: master.tex [hunk 6]
```diff
@@ -109,11 +128,12 @@
...
 %\input{scholarships}
 %\input{getting-around}
+
 \input{table-saturday}
 \input{table-sunday}
 \input{table-monday}
-\input{saturday}
-\input{sunday}
+%%%\input{saturday}
+%%%\input{sunday}
```
Comment (Nakaner) [inline master.tex:136]: Please remove the comments.

File: table-monday.tex [hunk 7]
```diff
@@ -5,136 +5,190 @@
...
 \pagestyle{monday-table}
 \setPageBackground
-\noindent\begin{landscape}
+\noindent%\begin{landscape}
   \begin{center}
-    \noindent\begin{tabular}{Z{0.75cm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}}
+
+    %% monday - morning %%
+
+    \makebox[8cm] {
```
Comment (Nakaner) [inline table-monday.tex:13]: Is `\makebox` necessary?

File: welcome.tex [hunk 8]
```diff
@@ -1,6 +1,6 @@
 \newpage
 \enlargethispage{1\baselineskip}
-\section*{Welcome to Heidelberg and State of the Map 2019} \label{welcome}
+\section*{Welcome to Heidelberg and State of the Map \numeral{2019}} \label{welcome}
```
Comment (Nakaner) [inline welcome.tex:3]: Please let it be as it is.

File: table-monday.tex [hunk 9]
```diff
@@ -5,136 +5,190 @@
...
+    \noindent\hspace*{1.3cm}\begin{tabular}[t]{Z{0.75cm}Z{3.6cm};{0.2mm/0.4mm}Z{3.6cm};{0.2mm/0.4mm}Z{1.0cm}}
       \cellcolor{commongray}
-      & \multicolumn{1}{c}{\cellcolor{GHS} GHS}
-      & \multicolumn{1}{c}{\cellcolor{HSO} HSO}
-      & \multicolumn{1}{c}{\cellcolor{KHS} KHS*}
-      & \multicolumn{1}{c}{\cellcolor{Mathematikon C} Mathematikon C*}
+      & \multicolumn{1}{c}{\cellcolor{GHS} Großer Hörsaal}
+      & \multicolumn{1}{c}{\cellcolor{HSO} Hörsaal Ost}
```
Comment (Nakaner) [inline table-monday.tex:17]: The booklet, the website and anything else using the Pretalx data use the abbreviations of the rooms. We cannot use the long names in the booklet because they take too much room on the description pages of the sessions.

**Review by Nakaner — CHANGES_REQUESTED** [review]
I don't understand why we invest so much time on 8 removed words and 1 changed word. The first  removal removed the obvious: The talk is about OSM. What else should it be about? The second removal removed the word "geospatial", an adjective to the term "remote sensing". Is there different remote sensing than geospatial remote sensing? The third change replace "through" by the "using". I admit that the last change is not necessary at all after I review my work a second time because I could change the width of the columns slightly to make "through" fit into the line.

The booklet is produced for the reader, not as a very official publication! Overview tables of the schedule are designed to make the readers able to decide quickly which talks are interesting. The less important a bit of information is the later you will find it in the booklet. The exact reproduction of submitted contents is not the aim of the tables. If it were, I would not even be able to replace "OpenStreetMap" by its abbreviation "OSM" which I did a lot and **even your pull request does**. Please treat these pages as an overview. I intentionally added footnotes too all shortened titles. I offer to add the following explanating sentence for the diamond symbol to EVERY DOUBLE PAGE of the schedule (it's explained at page 2 currently) and to add the page number where more details about the talks on that double page can be found:
> Title was shortened due to limited space. The original title can be found at pages \ref{abstracts}~et seqq.

The pull request a good starting point but there needs still some work to be done:

* The tables collides with the (currently green) boxes of the weekday. This should not be the case.
* The tables should respect the margins (except they are ugly, to be proven).
* Day labels on the right page or rotated by 180°. Please use the correct pair of pagestyles.
* Remove the diamond symbol entirely from the document if it is not used.

This won't surprise you: I don't like the table you propose (but will accept the pull request if all my issues mentioned above and in the inline comments are addressed) due to the following reasons:

* It contains too much whitespace.
* The white space after the last talk on Monday does not look nice. My solution has an empty page there as well but I plan to fill it with an non-commercial advertisment (local chapter, video recording or something like that).
* It requires TeXnical hacks (manually added vertical space). This will cause trouble when last-minute updates are required to the tables.

**Comment by Nakaner** [comment]
You can skip further work on this pull request because you would have to update the schedule (see commits on master branch) and because I succeeded to use the long titles of the three academic talks you complain about.

**Comment by tyrasd** [comment]
> "OpenStreetMap" by its abbreviation "OSM" which I did a lot and even your pull request does. 

Sorry, I must have overlooked these. I will replace these abbreviations as well. But also note that using well known abbreviations is quite a bit different from omitting significant portions of a title.

> The tables collides with the (currently green) boxes of the weekday. This should not be the case.

yeah, I would propose to move the weekday boxed to the top

> The tables should respect the margins (except they are ugly, to be proven).

Sorry, I don't understand this sentence. Can you explain what you mean again?

> Day labels on the right page or rotated by 180°. Please use the correct pair of pagestyles.

When the weekday boxes are at the top, this should be solved.

> Remove the diamond symbol entirely from the document if it is not used.

Jep, will do.

> The white space after the last talk on Monday does not look nice. My solution has an empty page there as well but I plan to fill it with an non-commercial advertisment (local chapter, video recording or something like that).

Can't we fill the whitespace in our solution in a similar way? E.g. a horizontal banner that spans the two pages?

> It requires TeXnical hacks (manually added vertical space). This will cause trouble when last-minute updates are required to the tables.

Yes, it's a bit hacky. But it's not that hard to align the two tables because the only thing you have to do is to add additional linebreaks at the slots with fewer lines than the corresponding entries on the other page.

> You can skip further work on this pull request because you would have to update the schedule (see commits on master branch) and because I succeeded to use the long titles of the three academic talks you complain about.

Excuse me?

I believe the two-page layout proposed here should still be considered. It has many benefits over the old layout: For example that not only the academic talks have full titles on the overview, that reading the program table doesn't require one to turn around ones head or the booklet, it doesn't mis-represent the 09:00 academic talk as a keynote, etc.

File: abstract-commands.tex (hunk 1 above)
Comment (tyrasd) [reply abstract-commands.tex:234]: I believe it's better to use color coding only for a single property, and not to mix up two different things: either we color-code rooms, or we color-code the different tracks

File: master.tex (hunk 2 above)
Comment (tyrasd) [reply master.tex:2]: sorry, this was a left-over from something I tried out earlier. I will get rid of it.

File: master.tex (hunk 3 above)
Comment (tyrasd) [reply master.tex:1]: sorry, this was a left-over from something I tried out earlier. I will revert it.

File: master.tex (hunk 4 above)
Comment (tyrasd) [reply master.tex:9]: I think it is better to use Barlow/D-Din fonts consistently as it was originally proposed with our logo and design style guidelines. We also use Barlow on the Website. I believe it is well readable. The D-Din font is according to it only used for the year number "2019" in the context of "State of the Map 2019".

File: master.tex (hunk 5 above)
Comment (tyrasd) [reply master.tex:62]: sorry, was a leftover from my earlier tests. I will remove it

File: master.tex (hunk 6 above)
Comment (tyrasd) [reply master.tex:136]: sure

File: table-monday.tex (hunk 7 above)
Comment (tyrasd) [reply table-monday.tex:13]: yeah, the table on the left side was shifted down a bit (I believe because it has some overhang over the page boundary). The makebox here was a quick workaround to make both tables be aligned.

File: welcome.tex (hunk 8 above)
Comment (tyrasd) [reply welcome.tex:3]: see my answer [above](https://github.com/osmfoundation/sotm2019-booklet/pull/1#discussion_r313829333)

File: table-monday.tex (hunk 9 above)
Comment (tyrasd) [reply table-monday.tex:17]: This is just a proposal. The room name abbreviations "GHS", "HSx", "KHS" are only informal. At the location, only the full names are signed on the room entrances. We can still change it on the website, pretalx and app(s), but in the booklet we need to decide now how we want to have it. I think it is better to print the actual room names.

**Comment by Nakaner** [comment]
> I believe the two-page layout proposed here should still be considered. It has many benefits over the old layout: For example that not only the academic talks have full titles on the overview, that reading the program table doesn't require one to turn around ones head or the booklet, it doesn't mis-represent the 09:00 academic talk as a keynote, etc.

Months ago the academic programme comittee planned to have a keynote. The topic of the talk looks very much like a keynote. It is possible with my layout to represent it as a normal talk (just tried it successfully).

**Comment by yair-grinb** [comment]
Sorry for my late contribution to this topic so far, relocation issues had left me little time for anything else. @tyrasd thank you for all the hard work, @Nakaner thank you for looking into this and commenting where needed.

> You can skip further work on this pull request because you would have to update the schedule (see commits on master branch) and because I succeeded to use the long titles of the three academic talks you complain about.

I do not agree with this view. One of the things mentioned during the Mumble talk is that the requests of the Academic Track come at the expense of the general track talks which titles also get shortened. The effects are quite noticeable, for example the title "OSMF local chapters in countries of the Global South what can we learn from OSM associations dynamics in French-speaking southern countries of Africa and the Caribbean ?" is shortened to something like "OSMF local chapters in countries of the Global South" if memory serves me right. These two titles lead to very different expectations and could cause disappointment for both the speakers and the audience.  

The solution proposed by @tyrasd allows including the full titles for everyone, so the request of one group do not require another group (unconscionably) giving up their rights. I also agree with @tyrasd that it makes the programme more accessible. Hence, I think we should still consider this new design.

**Comment by ChristineKarch** [comment]
I think we should stop this discussion now. The Academic peole has what they wanted and there is no need to continue a discussion on things they even don't belong to. Thank you Michael, for the effort you made to satisfy this strict requirements  by the them. I know about the hard work you did the last days but also the weeks before where you already know about these requirements and where already tried a lot of ways to make the academic titles as much complete as possible! 

@yair-grinb I respect your point of you, but let us stop now the discussion and go on with work on booklet. There are much more tasks for Michael than just the layout of this table.

**Comment by MarcoMinghini** [comment]
Dear @ChristineKarch, 
I acknowledge that we have many other important things to do for the conference, and I am fine to close this discussion. However, before this is done I would just like to clarify my doubts about the governance of our team. 

With reference to this:

> The Academic peole has what they wanted and there is no need to continue a discussion on things they even don't belong to.

personally, in this context I see myself first as a member of the SotM team and then as an academic person. I believe the same is for @yair-grinb and the same is for the local team, whose intervention (and the pull request) probably means they also would like to bring their position into the discussion.

So, I probably need just a clarification on what is the governance of the team, which also implies clarifying what is the role of the local team. Maybe a document already exists where I can find this information. Then I'm happy to close the discussion.

**Comment by ChristineKarch** [comment]
Michael has the responsibility to prepare the booklet. It is ok to give him input, and it is ok to force him to implement special needs (like the academic needs about complete titles in the schedule table) - to a certain degree (not endless). 

The academic part of the schedule was a long discussion and I know about the efforts Michael did. I (as chair of the working group) want to give him the possibility to go on with his work. He has still a lot of pages to do. So I finished this at a point where the academic requirements are satisfied.
